### PR TITLE
Preliminary fix for dot corruption in pic_filename

### PR DESCRIPTION
**Pull request by i92guboj**
Please, review it. It's only preliminary, I think we can simplify a bit in a couple of places, but first I'd like to see if this, indeed, fixes the issue at hand.

### CODE REVIEW

File: application/controllers/Items.php [hunk 1]
```diff
@@ -64,14 +64,23 @@ public function search()
...
+				$this->_update_pic_filename($item);
+			}
 		}
 
 		echo json_encode(array('total' => $total_rows, 'rows' => $data_rows));
 	}
 	
 	public function pic_thumb($pic_filename)
 	{
+	    if($pic_filename=='')
```
Comment (daN4cat) [inline application/controllers/Items.php]: can you please fix this and use tabs

File: application/controllers/Items.php [hunk 2]
```diff
@@ -216,9 +225,9 @@ public function view($item_id = -1)
 			// redundant.
 			if(sizeof($images) > 0)
```
Comment (daN4cat) [inline application/controllers/Items.php]: this doesn't seem required anymore, please remove the whole statement

File: application/controllers/Items.php [hunk 3]
```diff
@@ -809,6 +818,8 @@ public function do_excel_import()
 	 */
 	private function _update_pic_filename($item)
 	{
+        $filename = pathinfo($item->pic_filename, PATHINFO_FILENAME);
```
Comment (daN4cat) [inline application/controllers/Items.php]: tabs

**Comment by i92guboj** [comment]
That second commit should fix spacing (I corrected a few more spacing issues while I was at it). And removed some code. A third commit will remove one of the checks, that, at this stage, I think it's unnecessary.

I'll triple test everything, please, do your checks as well before merging.

**Pull request merged**